eclass/npm.eclass

```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2013 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+# @ECLASS: npm.eclass
+# @MAINTAINER:
+# Max Harley <augeocacher@gmail.com>
```

Comment:
Has this eclass not been updated in long enough that you should be the maintainer?